content/docs/command-reference/remote/modify.md

```diff
@@ -294,6 +294,41 @@ $ dvc remote modify myremote endpointurl \
 For more information on configuring Azure Storage connection strings, visit
 [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
 
+**Advanced Azure Authentication**
+
+If you rely on AAD to authenticate against your azure storage account, 
+the dvc azure remote supports a number of authentication methods. 
+The methods below enable authentication however you must
+set the storage account your container path exists in your dvc config or as an environment variable
+
+
+```
+$ dvc remote modify myremote storage_account 'my-storage-account'
+```
+
+Or
+
+```
+export AZURE_STORAGE_ACCOUNT='my-storage-account'
+```
+
+- `azcli_credential` - If you are currently logged in using `az cli` you can choose use this to authenticate with the remote
+  ```
+  $ dvc remote modify myremote azcli_credential True
+  ```
+
+- If using a service principal you can set the environment variables 
+  `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_TENANT_ID` and they will be picked up
+  
+  Or
+  
+  You can add the following variables into the config
+  ```
+  $ dvc remote modify myremote client_id 'my-client-id'
+  $ dvc remote modify myremote client_secret 'my-client-secret'
```

Comment:
I guess you'd want to keep the client secret and client id in your local configuration. So I'd suggest to also document it this way (with the `--local` attribute):
`dvc remote modify --local myremote client_id 'my-client-id'`

Reply:
```suggestion
  $ dvc remote modify --local myremote client_secret '<my-secret>'
```